lib/generators/declare_schema/migration/migrator.rb

```diff
@@ -521,6 +507,22 @@ def change_foreign_key_constraints(model, old_table_name, to_rename)
           [drop_fks + add_fks]
         end
 
+        def foreign_key_changes_due_to_column_renames(fks_to_drop, fks_to_add, to_rename)
+          fks_to_drop.each_with_object([[], []]) do |fk_to_drop, (renamed_fks_to_drop, renamed_fks_to_add)|
+            fk_to_add = fks_to_add.find do |fk_to_add|
+              fk_to_add.foreign_key.nil? and raise "Foreign key is not allowed to be nil for #{fk_to_add.inspect}"
```

Comment:
I think you can drop this assertion now that foreign keys are prohibited from being `nil?`.

Reply:
Yes, that makes sense.